python/cuml/test/test_preproc_utils.py

```diff
@@ -170,7 +170,7 @@ def sparsify_and_convert(dataset, conversion_format, sparsify_ratio=0.3):
     if conversion_format.startswith("cupy"):
         dataset = cp.asnumpy(dataset)
 
-    return cpu_csr_matrix(dataset), converted_dataset
+    return [cpu_csr_matrix(dataset), converted_dataset]
```

Comment:
Why use a list here?

Reply:
The array gets edited [here](https://github.com/rapidsai/cuml/blob/8bdc77f2232961055ae8a565830837366ef5b80e/python/cuml/test/test_preproc_utils.py#L251). A tuple doesn't allow this. I'll modify this to get a better code.